charts/umbrella/README.md

```diff
@@ -93,9 +131,8 @@ If you still face DNS issues afterwards, add the hosts to your /etc/hosts file:
 192.168.49.2    semantics.tx.test
 ```
 
-Replace 192.168.49.2 with your minikube ip.
-
-**Additional network setup** (for Mac only)
+#### Mac
+**Additional network setup**
```

Comment:
Please call this section Additional network setup for Mac and move it above the Windows section into the Linux and Mac section